src/gluonts/mx/trainer/_base.py

```diff
@@ -97,12 +99,19 @@ class Trainer:
     init
         Initializer of the weights of the network (default: "xavier").
     hybridize
-        If set to true the network will be hybridized before training
-    post_initialize_cb
-        An optional callback function. If provided the function will be called with the
-        initialized network `post_initialize_cb(net)` before the training starts.
-        This callback can be used to e.g. overwrite parameters for warm starting, to freeze some
-        of the network parameters etc.
+        If set to True the network will be hybridized before training
+    callbacks
+        A list of gluonts.mx.trainer.callback.Callback's or a CallbackList to control the training.
+    add_default_callbacks
+        bool, True by default. If True LearningRateReduction and ModelAveragingCallbacks are used in addition to the callbacks specified in the callbacks argument.
+        default callbacks:
+        >>>[ModelAveraging(avg_strategy=SelectNBestMean(num_models=1)),
+        ...LearningRateReduction(
+        ...base_lr=learning_rate,
+        ...decay_factor=learning_rate_decay_factor,
+        ...patience=patience,
+        ...min_lr=minimum_learning_rate,
+        ...objective="min")]
```

Comment:
I think these lines in the docstring are messing up with the docs build maybe we can summarize this differently